packages/web-components/fast-element/docs/guide/defining-elements.doc.md

```diff
@@ -495,14 +511,12 @@ Some context properties are opt-in because they are more costly to update. So, f
 
 #### Composing Templates
 
-The `HTMLTemplate` returned from the `html` tag helper is also a directive itself. As a result, you can create templates and compose them into other templates.
+The `ViewTemplate` returned from the `html` tag helper has special handling when it is used inside of another template. The is that you can create templates and compose them into other templates.
```

Comment:
> **The** is that you can create templates and compose them into other...


Should "the" be "this"? Phrasing seems off.

Reply:
Definitely off 🤣  I'll fix it.